airbyte-webapp/src/components/TextWithHTML/TextWithHTML.tsx

```diff
@@ -1,30 +1,24 @@
 import React from "react";
-import sanitizeHtml from "sanitize-html";
+import DOMPurify from "dompurify";
 
 type IProps = {
   text?: string;
 };
 
-const allowedAttributes = {
-  ...sanitizeHtml.defaults.allowedAttributes,
-  a: [...sanitizeHtml.defaults.allowedAttributes["a"], "rel"],
-};
-
 const TextWithHTML: React.FC<IProps> = ({ text }) => {
   if (!text) {
     return null;
   }
 
-  const sanitizedHtmlText = sanitizeHtml(text, {
-    allowedAttributes,
-    transformTags: {
-      a: sanitizeHtml.simpleTransform("a", {
-        target: "_blank",
-        rel: "noopener noreferrer",
-      }),
-    },
+  DOMPurify.addHook("afterSanitizeAttributes", function (node) {
```

Comment:
This registers a global event listener to the `DOMPurify` library on every render of this component. This is most likely not what we want, and we should make sure this event lister is only registered once, or find another way in DOMPurify to solve that problem.